libs/perun/openapi/src/lib/api.module.ts

```diff
@@ -49,7 +49,7 @@ import { VosManagerService } from './api/vosManager.service';
     VosManagerService ]
 })
 export class ApiModule {
-    public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders<ApiModule> {
+    public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders {
```

Comment:
Please, remove this change. I dunno why but this always happens after openapi generation and it is needed to specify the ApiModule generic type.

Reply:
I see. Removed.